spec/services/view_handover_email_addresses_summary_spec.rb

```diff
@@ -48,50 +48,11 @@
     end
   end
 
-  context 'with an offender that has case information but no team link' do
-    let(:offenders) { [double(offender_no: 1)] }
-
-    before { create(:case_information, nomis_offender_id: 1, team: nil) }
-
-    it 'returns the right counts' do
-      expect(result).to eq(
-        has_email_address: 0,
```

Comment:
As above: we can safely delete this entire service object and its associated rake task. It was never needed, and has never been used.